### PR TITLE
AO3-4762 added test for controllers/invite_requests_controller (admin deleting someone from queue)

### PR DESCRIPTION
**Pull request by pixelred**
## Issue

https://otwarchive.atlassian.net/browse/AO3-4762

### CODE REVIEW

File: features/other_a/invite_queue.feature [hunk 1]
```diff
@@ -35,6 +35,13 @@ Feature: Invite queue management
     When I follow "Get an Invitation"
     Then I should see "Request an Invitation"
 
+  Scenario: An admin can delete people from the queue
+    Given an invitation request for "invitee@example.org"
+      And I am logged in as an admin
+      And I go to the manage invite queue page
```
Comment (sarken) [inline features/other_a/invite_queue.feature]: Just a tiny nitpick here! Cucumbers usually have Given steps for setting up background, When steps for performing actions, and Then steps for describing results (https://cucumber.io/docs/reference#steps). It ultimately doesn't change how anything works, but for style, I'd suggest making this `When I go to the manage invite queue page` instead.

File: features/other_a/invite_queue.feature [hunk 2]
```diff
@@ -35,6 +35,13 @@ Feature: Invite queue management
...
+      And I follow "Delete"
+    Then I should see "Request was removed from the queue."
```
Comment (sarken) [inline features/other_a/invite_queue.feature]: This is good! I'd just suggest adding one more thing to make sure we're ending up on the right page: `And I should be on the manage invite queue page` Doing that should cover the redirect on line 53 of the controller.

**Comment by pixelred** [comment]
that makes sense :v: thank you for the help!

how does it look?

**Comment by sarken** [comment]
The steps look good! But one more style comment: Given, When, and Then steps should all be indented the same amount, and the And steps should be indented two spaces deeper. (It might be clearer if you look at the other scenarios in this file to see what I mean--I'm on my phone and can't illustrate at the moment.)

**Comment by pixelred** [comment]
ahh you're right. i see it now. sorry about that /o\

**Comment by pixelred** [comment]
oh; i also forgot the spacing between the scenario and the rest... i'll do that as soon as i'm on my computer

**Comment by pixelred** [comment]
whoops... thought i added the space this morning...

**Pull request merged**